repos/spack_repo/builtin/packages/py_cylc_uiserver/package.py

```diff
@@ -1,46 +0,0 @@
-# Copyright Spack Project Developers. See COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack_repo.builtin.build_systems.python import PythonPackage
-
-from spack.package import *
-
-
-class PyCylcUiserver(PythonPackage):
```

Comment:
AFAICT the only reason this is removed is the dependency on py-graphene-tornado. Given that and my comment on py-graphene-tornado, I think we should keep this package for now.